cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -16,33 +16,25 @@
                     Set, Tuple, Union)
 import numpy as np
 from matplotlib import pyplot as plt
-from cirq import devices, ops, circuits, sim, work
+from cirq import circuits, devices, ops, protocols, sim, work
 
 CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
                                                    ('xeb_fidelity', float)])
 
 
+@protocols.json_serializable_dataclass(frozen=True)
 class CrossEntropyResult:
-    """Results from a cross-entropy benchmarking (XEB) experiment."""
-
-    def __init__(self, cross_entropy_pairs: Sequence[CrossEntropyPair]):
-        """
-        Args:
-            cross_entropy_pairs: A sequence of NamedTuples, each of which
-                contains two fields: num_cycle which returns the circuit
-                depth as the number of cycles and xeb_fidelity which returns
-                the XEB fidelity after the given cycle number.
-        """
-        self._data = cross_entropy_pairs
-
-    @property
-    def data(self) -> Sequence[CrossEntropyPair]:
-        """Returns a sequence of CrossEntropyPairs.
-
-        Each CrossEntropyPair is a NamedTuple that contains a cycle number and
-        the corresponding XEB fidelity.
-        """
-        return self._data
+    """Results from a cross-entropy benchmarking (XEB) experiment.
+
+    Attributes:
+        data: A sequence of NamedTuples, each of which
+            contains two fields: num_cycle which returns the circuit
+            depth as the number of cycles and xeb_fidelity which returns
```

Comment:
nit: "as the number of cycles" explains nothing unless "cycle" itself is defined. I've seen the term being used for both a single layer of gates and for two layers: single-qubit gates following by two-qubit gates. Probably worth commenting on this explicitly.

Reply:
Done.